src/snyk/common/configuration/configuration.ts

```diff
@@ -196,16 +196,15 @@ export class Configuration implements IConfiguration {
   get isFedramp(): boolean {
     if (!this.customEndpoint) return false;
 
-    // FEDRAMP URL e.g. https://api.fedramp.snykgov.io
+    // FEDRAMP URL e.g. https://api.feddramp.snykgov.io
```

Comment:
Typo in the comment?

Reply:
I think just pointing out that the fedramp part is irrelevant (the domain is the trigger)

Reply:
Sorry yes, we check for `snykgov.io` only now